lib/finch-pipeline-app-stage.ts

```diff
@@ -81,5 +83,16 @@ export class FinchPipelineAppStage extends cdk.Stage {
     new PVREReportingStack(this, 'PVREReportingStack', { terminationProtection: true });
     
     new SSMPatchingStack(this, 'SSMPatchingStack', { terminationProtection: true });
+
+    UBUNTU_FLEET_ARCHS.forEach((arch) => {
+      new UbuntuCodeBuildStack(this, `UbuntuCodeBuildStack-${arch}`, {
```

Comment:
Also, `_` are not allowed in stack names, and both of our arches have `_` in them